.vscode/settings.json

```diff
@@ -3,10 +3,15 @@
     "python.linting.flake8Enabled": true,
     "python.linting.enabled": true,
     "data.preview.create.json.schema": false,
+    "data.preview.theme": "light",
+    // this doesn't work, don't know why yet
```

Comment:
In case helpful - my guess is that it's because `arrow` is not listed under the known language identifiers at the bottom of this page [here](https://code.visualstudio.com/docs/languages/identifiers). Maybe you can just copy and amend the second code chunk from that page to define the arrow id - or whether there's a VSCode extension for arrow that already provides this, I don't know.